addon/ng2/tasks/build-webpack-watch.ts

```diff
@@ -5,12 +5,12 @@ import * as webpack from 'webpack';
 import * as ProgressPlugin from 'webpack/lib/ProgressPlugin';
 import { NgCliWebpackConfig } from '../models/webpack-config';
 import { webpackOutputOptions } from '../models/';
-import { ServeTaskOptions } from '../commands/serve';
+import { BuildOptions } from '../commands/build';
 
 let lastHash: any = null;
 
 module.exports = Task.extend({
-  run: function(runTaskOptions: ServeTaskOptions) {
+  run: function(runTaskOptions: BuildOptions) {
```

Comment:
Strange, that nobody has noticed yet that `build-webpack` and `build-webpack-watch` tasks uses the options interface from `serve` command. :confused: